### ISSUE TITLE
A11y - Home page - hero section - text contrast on image

### ISSUE THREAD

**Issue by luistoptal** [gigascience/gigadb-website#1366]
## User story

Text over image in hero section should have enough contrast and be readable

![image](https://github.com/gigascience/gigadb-website/assets/143437854/fcd2f068-efc7-41fa-865e-dca5a92984c6)

## Acceptance criteria

- [x] When text is presented over a background image, the text must have a background color defined (typically in CSS) that provides adequate text contrast when the background image is disabled or unavailable
- [x] Visually, text readability should be apparent and have a color contrast equivalent to at least 4.5/1 between text and background

## Additional Info

https://webaim.org/standards/wcag/checklist#sc1.4.3

**Comment by luistoptal** [comment]
Related branch here https://github.com/luistoptal/gigadb-website/tree/new-feature/1366/home-page-hero-section-text-contrast-on-image

**Comment by only1chunts** [comment]
@luistoptal can this issue be closed?
